_includes/header.html

```diff
@@ -3,8 +3,9 @@
     <a href="{{ site.github.url }}/" class="logo">{{ site.data.settings.title }}</a>
   </div>
   <nav>
+    |
     {% for item in site.data.settings.menu %}
-      <a href="{{ site.github.url }}/{{ item.url }}">{{ item.name }}</a>
+      <a href="{{ site.github.url }}/{{ item.url }}">{{ item.name }}</a> | 
```

Comment:
nit: no space needed after | at the end of this line, or alternatively can place `|` by itself on the following line.

Reply:
fixed